tests/ctl/core/test_api.py

```diff
@@ -1665,6 +1665,40 @@ def test_list_with_pagination_and_multiple_filters_2(
 
         assert result_json["items"][0]["fides_key"] == tcf_system.fides_key
 
+    @pytest.mark.parametrize(
+        "vendor_deleted_date, expected_systems_count, show_deleted",
+        [
+            (datetime.now() - timedelta(days=1), 1, True),
+            (datetime.now() - timedelta(days=1), 0, None),
```

Comment:
nit - reads a bit weird to me that the value here is `True` or `None`, although i know that `None` is falsey 😛 
```suggestion
            (datetime.now() - timedelta(days=1), 0, False),
```